### DISCUSSION TITLE
Run from new release pipeline but compare against existing pipeline

### DISCUSSION THREAD

**Discussion by mreyeros** [rfennell/AzurePipelines#1975] [Q&A]
I have an existing release pipeline for deploying my app/code to azure. I wanted to know if I could create a new release pipeline that looks at the last good release from the existing pipeline and compares it against the build that triggered the new pipeline.  What I want to do is create my release notes prior to actually running the existing pipeline to give it a once over to ensure that all PBI are accounted for in the new release.  I tried setting it up and overriding the release Id but it looks like it scans all builds since the beginning of time and takes over an hour to produce the release notes so I assume that it is not actually comparing against the last successful release.  Is it possible to do what I am trying to do?

**Comment by rfennell** [comment]
Sorry no, you can only compare runs based on the same base definition irrespective of whether is multistage YAML or Classic release
